thriftbp/merger.go

```diff
@@ -1,17 +1,15 @@
 package thriftbp
 
-import (
-	"github.com/apache/thrift/lib/go/thrift"
-)
-
-// Merge merges together the specified processors.
+// Merge merges together multiple processors into the first one.
 //
-// It's useful if the server need to support two separated thrift files.
+// It's useful when the server need to support more than one separated thrift
+// files.
```

Comment:
I think this should be “file”?

Reply:
`more than one` so it should be plural?

Reply:
What about “more than one thrift file”?

Reply:
oh so you are thinking about "more than (one thrift file)" and I'm thinking about "(more than one) thrift files".

Reply:
done.